lerna.json

```diff
@@ -4,7 +4,6 @@
   "commands": {
     "publish": {
       "ignore": [
-        "DEPENDENCIES.md",
         "*.tgz"
```

Comment:
Can this be removed like https://github.com/cerner/terra-core/pull/2548/files#diff-52d4bf43db96176e77cc9038a78f98e1 ?

Reply:
Updated: https://github.com/cerner/terra-framework/pull/814/commits/1a3a1271b2d65b092123e2ad62d7e37313541b4e